wdl-analysis/Cargo.toml

```diff
@@ -32,7 +33,6 @@ tokio = { workspace = true }
 tracing = { workspace = true }
 url = { workspace = true }
 uuid = { workspace = true, features = ["v4"] }
-walkdir = { workspace = true }
```

Comment:
`walkdir` is used in `wdl-engine`. Should it be replaced there with the `ignore` crate so we can drop this dep? Or is that not worth dealing with?